source/base/DateTime.ooc

```diff
@@ -41,6 +41,7 @@ DateTime: cover {
 	_ticks: ULong = 0
 	ticks ::= this _ticks
 	init: func@ (=_ticks)
+	init: func@ ~fromNanoseconds (nanoseconds: ULong) { this _ticks = nanoseconds / This nanosecondsPerTick }
```

Comment:
Not really, the goal is to abstract what we use internally. It should be easy to change the definition of the backend counter without breaking the code.